tracer/src/Datadog.Trace/Telemetry/TelemetryControllerV2.cs

```diff
@@ -229,6 +230,8 @@ private async Task PushTelemetry(bool sendAppClosing)
     {
         try
         {
+            CollectIastMetrics();
```

Comment:
I don't know, this doesn't feel right to me 🤔 I feel like the Iast metrics should be just like normal metrics, not tacked on here...

Reply:
That is the problem of having the metrics detected in the native side... We do not access directly the managed metrics and counts, we need to ask the native side every hearbeat.